rust/worker/src/execution/operators/merge_knn_results.rs

```diff
@@ -11,13 +11,18 @@ use thiserror::Error;
 #[derive(Debug)]
 pub struct MergeKnnResultsOperator {}
 
+#[derive(Debug)]
+pub struct MergeKnnBruteForceResultInput {
+    pub user_ids: Vec<String>,
+    pub distances: Vec<f32>,
+    pub vectors: Vec<Vec<f32>>,
+}
```

Comment:
added a separate struct here instead of using `BruteForceKnnOperatorOutput` to keep operator isolation, open to using `BruteForceKnnOperatorOutput` instead

Reply:
I would be fine personally with strongly coupling these things as it avoids a copy - the operators are rather strongly coupled as is - its a bit pretend that they are multi purpose. 

We need to rethink / clean up how we move data around in general

Reply:
sg, it's not copying so will leave as-is for now